### PR TITLE
Create filtering data grid component

### PR DESCRIPTION
**Pull request by paulagutierrez08**
Add filtering props on the DataGrid Component

image

### CODE REVIEW

File: src/atoms/data-grid.tsx [hunk 1]
```diff
@@ -1,19 +1,25 @@
 import React from 'react';
```
Comment (andrea-carmona) [inline src/atoms/data-grid.tsx:1]: espacio entre linea

File: src/atoms/data-grid.tsx [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
 
 interface DataGridProps extends Omit<MuiDataGridProps, 'columns' | 'rows'> {
   columns: GridColDef[];
+  disableColumnFilter: boolean;
+  filterModel?: GridFilterModel;
   rows: Record<string, unknown>[];
 }
 
-function DataGrid({ columns, rows, ...other }: DataGridProps) {
+function DataGrid({ columns, rows, disableColumnFilter, filterModel, ...other}: DataGridProps) {
```
Comment (juanav22) [inline src/atoms/data-grid.tsx]: Engineer, arrange the props in alphabetical order

**Review by andrea-carmona — APPROVED** [review]
Looks good foe me

**Review by juanav22 — APPROVED** [review]
Good  job!

**Review by andrea-carmona — APPROVED** [review]

**Comment by andrea-carmona** [comment]
Really good job !

**Pull request merged**